src/test/java/edu/oakland/test/database/DatabasePersistentStorageTest.java

```diff
@@ -0,0 +1,326 @@
+package edu.oakland.test.database;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import edu.oakland.production.database.DatabasePersistentStorage;
+import edu.oakland.production.database.DatabasePersistentStorageClass;
+import java.lang.IllegalArgumentException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Random;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Database PersistentStorage Test")
```

Comment:
For sake of standardization to make it easier to read and search the logs, can we change this to `DatabasePersistentStorage Unit Tests`? Thank you!
```suggestion
@DisplayName("DatabasePersistentStorage Unit Tests")
```